drivers/clock_control/clock_stm32f2_f4_f7.c

```diff
@@ -25,30 +24,61 @@
 #define z_pllp(v) LL_RCC_PLLP_DIV_ ## v
 #define pllp(v) z_pllp(v)
 
+/**
+ * @brief Return PLL source
+ */
+__unused
+static uint32_t get_pll_source(void)
+{
+	if (IS_ENABLED(STM32_PLL_SRC_HSI)) {
+		return LL_RCC_PLLSOURCE_HSI;
+	} else if (IS_ENABLED(STM32_PLL_SRC_HSE)) {
+		return LL_RCC_PLLSOURCE_HSE;
+	}
+
+	return -ENOTSUP;
+}
+
 /**
  * @brief Set up pll configuration
  */
+__unused
 int config_pll_sysclock(void)
 {
-	uint32_t pll_source, pll_m, pll_n, pll_p;
-
-	pll_n = STM32_PLL_N_MULTIPLIER;
-	pll_m = pllm(STM32_PLL_M_DIVISOR);
-	pll_p = pllp(STM32_PLL_P_DIVISOR);
+	uint32_t pll_source;
 
-	/* Configure PLL source */
-	if (IS_ENABLED(STM32_PLL_SRC_HSI)) {
-		pll_source = LL_RCC_PLLSOURCE_HSI;
-	} else if (IS_ENABLED(STM32_PLL_SRC_HSE)) {
-		pll_source = LL_RCC_PLLSOURCE_HSE;
-	} else {
+	pll_source = get_pll_source();
+	if (pll_source < 0) {
 		return -ENOTSUP;
 	}
 
-	LL_RCC_PLL_ConfigDomain_SYS(pll_source, pll_m, pll_n, pll_p);
+	LL_RCC_PLL_ConfigDomain_SYS(pll_source,
+				    pllm(STM32_PLL_M_DIVISOR),
+				    STM32_PLL_N_MULTIPLIER,
+				    pllp(STM32_PLL_P_DIVISOR));
 
 	return 0;
 }
+
+/**
+ * @brief Return pllout frequency
+ */
+__unused
+uint32_t get_pllout_frequency(void)
+{
+	uint32_t pll_source;
+
+	pll_source = get_pll_source();
+	if (pll_source < 0) {
```

Comment:
`get_pll_source()` returns `uint32_t`.
Therefore, `if (pll_source < 0)` is always `false`.